code/game/turfs/simulated/floor/fancy_floor.dm

```diff
@@ -113,6 +114,32 @@
 		icon_state = "basalt[rand(0, 12)]"
 		set_basalt_light(src)
 
+
+/turf/open/floor/grass/fakebasalt //Heart is not a real planeteer power
+	name = "aesthetic volcanic flooring"
+	desc = "Safely recreated turf for your hellplanet-scaping"
+	icon = 'icons/turf/floors.dmi'
+	icon_state = "basalt"
+	floor_tile = /obj/item/stack/tile/basalt
+	ore_type = /obj/item/weapon/ore/glass/basalt
+	slowdown = 0
+
+/turf/open/floor/grass/fakebasalt/Initialize()
+	..()
+	if(prob(15))
+		icon_state = "basalt[rand(0, 12)]"
+		set_basalt_light(src)
+
+/turf/open/floor/grass/fakebasalt/attackby(obj/item/C, mob/user, params)
```

Comment:
Which explicitly? I don't really care to change it unless its a problem, since you can't dig up grass/snow/basalt turfs it made sense, and im still a amateur coder learning the ropes so im not sure it would have carried over correctly so having the grass interaction explicitly mentioned again was worthwhile just for my peace of mind.

Its copy-paste-modify code and it works as intended from playtesting in spitting out 2 ash with the spade interaction. I don't mind the extra work.